bin/crew

```diff
@@ -1308,6 +1308,9 @@ def resolve_dependencies
   # leave only dependency names (remove all package attributes returned by @pkg.get_deps_list)
   @dependencies.map!(&:keys).flatten!
 
+  # abort if we have incompatible dependencies
+  abort "Some dependencies are not compatible with your device architecture (#{ARCH}). Unable to continue.".lightred if @dependencies.any? { |dep| !Package.load_package("#{CREW_PACKAGES_PATH}/#{dep}.rb").compatible? }
```

Comment:
It might be helpful to list the incompatible dependencies.

Reply:
To do that would mean reworking the code to be slower and more complex. 

In this case, given that this is only here as a worst-case scenario backup, I think makes more sense not to.